src/common/obj_bencher.cc

```diff
@@ -22,6 +22,8 @@
 
 const std::string BENCH_LASTRUN_METADATA = "benchmark_last_metadata";
 const std::string BENCH_PREFIX = "benchmark_data";
+const std::string BENCH_OBJ_NAME = BENCH_PREFIX + "_%s_%d_object%d";
+
 static char cached_hostname[30] = {0};
```

Comment:
That's not related but HOST_NAME_MAX is 64 so using 30 is a little bit limited here. What is the reason of keeping only a part of the hostname ?

Reply:
1. It used to be like that (see diff to commit 42d81bffc4d57bceb98872dadc8e57400981588c)
2. Most hostnames are shorter than that anyway.

Reply:
static char cached_hostname[HOST_NAME_MAX] = {0};
As long as we do have a system variable for that, using seems more logical to me.

Reply:
I agree, but if we change that now, and someone actually has a 31+ chars hostname, rados bench rand/seq will break for them.

Reply:
I didn't got why :(

Reply:
Why it'll break? Benchmark metadata file contains only previous PID, object size and number of object. Once hostname changes - either because of admin actions, running on different machine or because it used to be 30 chars and now is 50, generated object names will be wrong.